core/src/main/java/org/apache/accumulo/core/conf/PropertyType.java

```diff
@@ -56,8 +56,10 @@
           + "Examples of valid host lists are 'localhost:2000,www.example.com,10.10.1.1:500' and 'localhost'.\n"
           + "Examples of invalid host lists are '', ':1000', and 'localhost:80000'"),
 
-  PORT("port", Predicates.or(new Bounds(1024, 65535), in(true, "0")),
-      "An positive integer in the range 1024-65535, not already in use or specified elsewhere in the configuration"),
+  @SuppressWarnings("unchecked")
+  PORT("port", Predicates.or(new Bounds(1024, 65535), in(true, "0"), new Matches("\\d{1,5}-\\d{1,5}")),
```

Comment:
The regex could be a bit more restrictive, like `\\d{4,5}-\\d{4,5}`

Would there be a benefit for creating a Predicate to verify the port range, like maybe fail sooner (when setting a prop in shell).  Instead of validating in getPorts().